docs/3_finetuner_in_action/3_3_text_to_image_using_clip.md

```diff
@@ -1,3 +1,107 @@
 # Text to image search using CLIP
 
-Coming soon!
\ No newline at end of file
+This guide will showcase fine-tuning a `CLIP` model for text to image retrieval.
+
+## Task overview
+We'll be fine-tuning CLIP on a fashion captioning dataset which contains information about fashion products.
+
+For each product the dataset contains a title and images of multiple variants of the product. We constructed a parent `Document` for each picture, which contains two chunks: an image document and a text document holding the description of the product.
```

Comment:
if a product has multiple images, will the image document be: `Document(chunks=[Document(image_0), Document(image_1])`?

Reply:
Document(chunks=[Document(image_0), Document(title)], Document(chunks=[Document(image_1), Document(title)], etc.